src/app/modules/landing/modules/auth/components/login/login.component.html

```diff
@@ -15,7 +15,7 @@
   <div class="col-lg-4 col-12">
     <div class="card">
       <div class="card-body">
-        <form [formGroup]="form" (keydown.enter)="submitForm()">
+        <form [formGroup]="form" (keydown.enter)="submitForm($event)">
```

Comment:
no need to send $event. you can create a flag in ts like this:
`submitClicked = false;`
When click, set it to true
In subscribe set it to false

in submitForm() method control it

if(!this.submitClicked) return;

this.submitClicked = true;

Reply:
Yes good idea, done :)